### PR TITLE
Update node-providers.md

### PR DESCRIPTION
**Pull request by konstantinrm**
**Description**

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Review by sbvegan — CHANGES_REQUESTED** [review]
Just a few updates and then I can merge this in. Thanks for your contribution!

File: community/node-providers.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
 # Node Providers
 
 Checkout the node and API providers that our community is using or has built! Please open a pull request if you'd like something to be added to this list.
+
+## DRPC.ORG
+
+### Description and Pricing
```
[the remark this answers is not in the document]
Comment (konstantinrm) [reply community/node-providers.md]: Done

File: community/node-providers.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
 # Node Providers
 
 Checkout the node and API providers that our community is using or has built! Please open a pull request if you'd like something to be added to this list.
+
+## DRPC.ORG
```
[the remark this answers is not in the document]
Comment (konstantinrm) [reply community/node-providers.md]: Done

File: community/node-providers.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+
+### Description and Pricing
+
+[DRPC](https://drpc.org/) provides auto-scalable Optimism RPC (Full & Archive) nodes and RPCs for 24+ other chains. DRPC supports Websocket and all methods, including debug and trace methods. For early-stage startups, DRPC, together with Optimism Foundation, provides OP Archive nodes from 3 geo clusters without methods restrictions and totally free! For commercial nodes, DRPC uses a pay-as-you-go model with payments per [CU - Compute units](https://docs.drpc.org/pricing/compute-units).
```
[the remark this answers is not in the document]
Comment (konstantinrm) [reply community/node-providers.md]: Fixed

**Review by sbvegan — APPROVED** [review]
Thanks for making those updates and contributing to our repo!

**Pull request merged**